backend/api/tests/test_models.py

```
@@ -0,0 +1,222 @@
+        assert str(profile1) == "Named Company"
+        
+        # Test the string representation without company name
+        user2 = User.objects.create_user(
+            email="company2@example.com",
+            firebase_uid="companyuid2",
+            role="company"
+        )
+        profile2 = CompanyProfile.objects.create(
+            user=user2
+        )
+        assert str(profile2) == f"Profile for company2@example.com"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)